docs/deploy.en-US.md

```diff
@@ -0,0 +1,141 @@
+---
+order: 8
+title: Build & Deploy
+type: Basic
+---
+
+## Build
+
+When you finish the development, you can run the following command to build your application:
+
+```bash
+$ npm run build
+```
+
+Since Ant Design Pro use [roadhog](https://github.com/sorrycc/roadhog) as development tool, complex processes have been encapsulated and for most scenarios only one command `roadhog build` is required to build the package, after build successfully, it will generate the `dist` folder in the root directory, which is contains packaged files, usually static files like` *** .js`, `***.css`, ` index.html`.
```

Comment:
...the `dist` folder will be generated in the root directory, which contains packaged files...